src/math/Vector3.js

```diff
@@ -5,87 +5,248 @@
  * @author mikael emtinger / http://gomo.se/
  * @author egraether / http://egraether.com/
  * @author WestLangley / http://github.com/WestLangley
+ * @author Ben Houston / https://clara.io
  */
 
-THREE.Vector3 = function ( x, y, z ) {
+THREE.Vector3 = function ( array, x, y, z ) {
+
+	if( typeof buffer === 'Number' ) {
```

Comment:
Primitive type is lowercase, so `typeof buffer === 'number'` whereas `boxedNumber instanceof Number`, but boxing is explicit so `boxedNumber = new Number( primitiveNumber )` otherwise won't be an instance of `Number`.